packages/cli/yaml/yaml-schema/src/docsAst/visitDocsConfigFileAst.ts

```diff
@@ -18,6 +23,38 @@ export async function visitDocsConfigFileYamlAst(
         []
     );
 
+    let pageEntries: [string, string][];
+
+    try {
+        const { pages } = await docsYml.parseDocsConfiguration({
+            rawDocsConfiguration: contents,
+            context,
+            absoluteFilepathToDocsConfig: absoluteFilepathToConfiguration,
+            absolutePathToFernFolder
+        });
+        pageEntries = Object.entries(pages);
+    } catch {
+        pageEntries = [];
+    }
+
+    for (const [relativePath, markdown] of pageEntries) {
+        const { filepaths } = parseImagePaths(markdown, {
+            absolutePathToFernFolder,
+            absolutePathToMdx: resolve(absolutePathToFernFolder, relativePath)
+        });
+
+        for (const filepath of filepaths) {
+            await visitor.filepath?.(
+                {
+                    absoluteFilepath: filepath,
+                    value: path.relative(absolutePathToFernFolder, filepath),
+                    willBeUploaded: true
```

Comment:
markdown files don't get uploaded right to s3

Reply:
This part of the code is looping over media files referenced in markdown, and running checks on their filepaths. It's my understanding that those are being uploaded to S3, so I've marked them as such. The markdown files themselves are handled elsewhere in this file.

Reply:
sorry i think the code collapsing got me confused. this makes sense!